examples/js/controls/TransformControls.js

```diff
@@ -903,7 +913,7 @@
 
 		function onPointerMove( event ) {
 
-			if ( scope.object === undefined || scope.axis === null || _dragging === false || ( event.button !== undefined && event.button !== 0 ) ) return;
```

Comment:
The `event.button` check was meant to avoid being able to drag with other mouse buttons than the primary one, because you'll often want to do other things with other buttons in your viewport. I don't know much about pointer events but should be safe to just re-add the check?